Misc/NEWS.d/next/Library/2023-04-22-21-34-13.gh-issue-103693.SBtuLQ.rst

```diff
@@ -0,0 +1 @@
+Add convenience variable feature to :mod:`pdb`
```

Comment:
Where did you get the name "convenience?"  I might have called it temporary variables.

Reply:
It was borrowed from `gdb` - https://ftp.gnu.org/old-gnu/Manuals/gdb/html_node/gdb_59.html